src/jit/flowgraph.cpp

```diff
@@ -22471,3 +22481,1010 @@ void Compiler::fgLclFldAssign(unsigned lclNum)
         lvaSetVarDoNotEnregister(lclNum DEBUGARG(DNER_LocalField));
     }
 }
+
+//------------------------------------------------------------------------
+// fgRemoveEmptyFinally: Remove try/finallys where the finally is empty
+//
```

Comment:
It might be useful to highlight in the function header what the current limitations are, e.g. only single basic block, not doing 'fault' regions, must contain GT_RETFILT, etc.